Modules/_testcapimodule.c

```diff
@@ -5856,6 +5856,97 @@ static PyTypeObject Generic_Type = {
 };
 
 
+/* Test various calling conventions. These functions always return a
+   two-element list [args, kwargs] */
+
+static PyObject *
+CallTest_call(PyObject *Py_UNUSED(self), PyObject *args, PyObject *kwargs)
+{
+    PyObject *res = PyList_New(2);
+    if (res == NULL) {
+        return NULL;
+    }
+    Py_INCREF(args);
+    PyList_SET_ITEM(res, 0, args);
+    if (kwargs == NULL) {
+        kwargs = PyDict_New();
+        if (kwargs == NULL) {
+            Py_DECREF(res);
+            return NULL;
+        }
+        PyList_SET_ITEM(res, 1, kwargs);
+    }
+    else {
+        Py_INCREF(kwargs);
+        PyList_SET_ITEM(res, 1, kwargs);
+    }
+    return res;
+}
+
+static PyObject *
+CallTest_varargs(PyObject *self, PyObject *args)
+{
+    return PyObject_Call(self, args, NULL);
+}
+
+static PyObject *
+CallTest_varargs_keywords(PyObject *self, PyObject *args, PyObject *kwargs)
+{
+    return PyObject_Call(self, args, kwargs);
+}
+
+static PyObject *
+CallTest_noargs(PyObject *self, PyObject *unused)
+{
+    if (unused != NULL) {
+        PyErr_BadInternalCall();
+        return NULL;
+    }
+    return _PyObject_CallNoArg(self);
+}
+
+static PyObject *
+CallTest_onearg(PyObject *self, PyObject *arg)
+{
+    return _PyObject_CallOneArg(self, arg);
+}
+
+static PyObject *
+CallTest_fastcall(PyObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    return _PyObject_Vectorcall(self, args, nargs, NULL);
+}
+
+static PyObject *
+CallTest_fastcall_keywords(PyObject *self, PyObject *const *args, Py_ssize_t nargs, PyObject *kwnames)
```

Comment:
PEP 7: Line too long,
```suggestion
CallTest_fastcall_keywords(PyObject *self, PyObject *const *args,
                           Py_ssize_t nargs, PyObject *kwnames)
```